tests/test_visitors/test_ast/test_loops/test_loops/test_infinite_while_loops.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.violations.best_practices import (
+    InfiniteWhileLoopViolation,
+)
+from wemake_python_styleguide.visitors.ast.loops import WrongLoopVisitor
+
+incorrect_loop1 = """
+while True:
```

Comment:
Looks like there are a lot of repetitive patterns in these samples.
Let's parametrize them!

Like so:

```python
incorrect_loop_template = """
while True:
    {0}
"""
```

And later we can use this way of testing:
```python
@pytest.mark.parametrize('keyword', [
    'break',
    'return',
    'return some',
    'raise Some',
    'raise Some()',
    'raise',
]
def test_whatever(keyword):
     code = template.format(keyword)
```

We also need to test a nested keyword: 

```python
while True:
    if some:
        {0}
```

Reply:
Alright I think I fixed everything! Let me know what you think!